slave/slave_channels.go

```diff
@@ -0,0 +1,43 @@
+package slave
+
+import (
+	"fmt"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+)
+
+func addTask(task lib.Task, s *Slave) {
```

Comment:
Agree with the receiver argument, will fix that. I will think about if I move the methods back into `slave.go`. I am just used to organizing code :P